### ISSUE TITLE
Update reference card for 3D Tiles 1.1

### ISSUE THREAD

**Issue by javagl** [CesiumGS/3d-tiles#703]
One point of https://github.com/CesiumGS/3d-tiles/issues/651 that had still been on my plate (and been waiting there for too long) is the update of the reference card for 3D Tiles 1.1. To avoid filling the repo with commits of SVG (or even worse, binaries like PDFs), certain points can probably be sorted out here.

I assume that the result should be a dedicated reference card for the new features of 3D Tiles 1.1. Specifically, I assume that it should _not_ combine the reference card of 1.0 and the reference card for 'Next'. Instead, it should rather be an update of the former 'Next' reference card. 

Two broader, structural questions:

- Where/how to present the S2 extension? 
  It could be a dedicated section at the end, to make clear that it is not part of "3D Tiles (core) 1.1". Or it might be omitted in the reference card altogether: It only was a short section to _present_ the extension, but in terms of contents, it was not adding much value beyond the main S2 README. 

- Where/how to present the glTF extensions? 
  There already is some confusion, and people think that the glTF extensions are somehow "part of" Next/1.1. Also, the glTF extensions have been split into `mesh_features` and `structural_metadata` in the meantime. One could mention `structural_metadata` as one implementations of the 3D Metadata Specification. But this should be done carefully, with a disclaimer: Technically, this extension is _not_ part of 3D Tiles. 
  For now, I think that it could make sense to omit the page "The EXT_mesh_features Extension" from the 'Next' reference card. This could be moved into a short (1-2 pages) but dedicated overview of the proposed `mesh_features` and `structural_metadata` extensions.

**Comment by lilleyse** [comment]
> I assume that the result should be a dedicated reference card for the new features of 3D Tiles 1.1. Specifically, I assume that it should _not_ combine the reference card of 1.0 and the reference card for 'Next'. Instead, it should rather be an update of the former 'Next' reference card. 

So it would be like a "what's new in 3D Tiles 1.1?" reference card? Even though I wasn't thinking that originally that's probably the right thing to do.

> Where/how to present the S2 extension?

It would be fine to omit

> Where/how to present the glTF extensions?

There could be a separate resource that describes all the recommended glTF extensions. Which is reminiscent of the glTF geospatial profile: https://github.com/KhronosGroup/glTF/issues/2092

**Comment by javagl** [comment]
Right now, we have the `/reference-card` subdirectory and the `/next/reference-card` subdirectory. The latter is supposed to be removed at some point. The updated reference card will contain what's new for 3D Tiles 1.1. 

Two options come to my mind:

- There could be a `/reference-cards` directory, with `1.0` and `1.1` as subdirectories
- There could be a `/reference-card-1.1` directory 

Also: I'd do this update in a state that is branched off from `draft-1.1`. (It _could_ be done in `master`, because it's largely independent of other changes, but think that doing it in the `draft-1.1` branch just makes more sense)

Here's the 'release candidate':

[3d-tiles-reference-card-1.1-2022-07-25-RC01.pdf](https://github.com/CesiumGS/3d-tiles/files/9184365/3d-tiles-reference-card-1.1-2022-07-25-RC01.pdf)

**Comment by lilleyse** [comment]
> There could be a `/reference-cards` directory, with 1.0 and 1.1 as subdirectories

This option seems good. Have you thought about where the PDFs would go and how the README's would be updated?

> I'd do this update in a state that is branched off from draft-1.1

:+1:

**Comment by javagl** [comment]
> Have you thought about where the PDFs would go and how the README's would be updated?

The current reference card PDF is in the root directory. I considered to move it to the `reference-cards` directory, but thought that there might be some direct links to this file out there. So I just added the 1.1 PDF in the root directory as well. 

The main README.md now links to the `reference-cards` subdirectory, and the `reference-cards/README.md` contains the preview images now. (They could both be added to the main README, but maybe it's better to keep the main README lean...)

Entry point from main README: https://github.com/javagl/3d-tiles/tree/draft-1.1-reference-card-update#3d-tiles-reference-cards

The PR is at https://github.com/CesiumGS/3d-tiles/pull/706

(This PR does **NOT** yet remove the `next` subfolder, although we might be approaching the point where this could be done)

**Comment by lilleyse** [comment]
> (This PR does **NOT** yet remove the `next` subfolder, although we might be approaching the point where this could be done)

Yeah, I think we can start that process now by removing `next` from the `draft-1.1` branch. The revision history and logos have served their purpose and are ok to remove (and will still be accessible in the git history if we need them). The rest of the content has migrated to the new resources page and reference card. I don't think there's anything left that needs to be kept.

Once `draft-1.1` is ready to merge into `main` we'll need to update any broken links that point to `next`.
